nep-5.mediawiki

```diff
@@ -71,6 +71,14 @@ The parameter <code>account</code> SHOULD be a 20-byte address. If not, this met
 
 If the <code>account</code> is an unused address, this method MUST return <code>0</code>.
 
+====onPayment====
+
+This method is optional, it will be called during NEP5 transfers in order to ensure that the smart contract know how to deal with the token, if don't this method SHOULD <code>throw</code> an exception.
```

Comment:
```suggestion
This method is optional, it will be called during NEP5 transfers to give the smart contract the opportunity to further process the event and optionally stop execution by throwing an exception.
```